console/printCell.c

```diff
@@ -1,11 +1,36 @@
 #include <stdio.h>
+#include <unistd.h>
 #include "../mySimpleComputer/sc_constants.h"
 #include "../mySimpleComputer/sc_variables.h"
+#include "../myTerm/mt_constants.h"
+#include "../include/myTerm.h"
+#include "../include/enums.h"
 
-void sc_printCell(int address)
+void printCell(int address, enum colors fg, enum colors bg)
 {
     if (address >= RAM_MIN_ADDRESS && address <= RAM_MAX_ADDRESS)
```

Comment:
Здесь лучше сделать проверку на не валидный адрес 
```suggestion
    if (address < RAM_MIN_ADDRESS || address > RAM_MAX_ADDRESS)
    {
      printf("ERROR. Invalid adress\n");
      return;
    }
```
Так будет меньше вложенность